tests/python/gaia-ui-tests/gaiatest/apps/phone/app.py

```diff
@@ -10,6 +10,7 @@
 class Phone(Base):
 
     name = "Phone"
```

Comment:
double quote here, perhaps we should make it consistent?  (not sure whether there is a difference though)